### DISCUSSION TITLE
Code Block document editor feature

### DISCUSSION THREAD

**Discussion by pbarnum** [Leantime/leantime#1109] [Ideas]
Hello all,

I'm not sure if this has been brought up before (not in my search results at least), but I was wondering if "Code Blocks" were available or planned to be implemented into the document/wiki editor?

`To be explicit`
```
This is what I mean.
Above is the "inline" code
And this is the "multi-line" code block
```
This feature is the only thing that is stopping me from using the `Documents/Wiki` tool since I will mostly use this for managing my personal programming projects.

Thank you!

**Comment by marcelfolaron** [comment]
Hi, 

Yes that should be pretty easy. Let me create an issue for that.
